roles/neutron-data-network/tasks/main.yml

```diff
@@ -59,6 +59,45 @@
     neutron-metadata-agent:
       config_files: /etc/neutron/metadata_agent.ini
 
+- name: neutron lbaas config
+  template:
+    src=etc/neutron/neutron_lbaas.conf
+    dest=/etc/neutron/neutron_lbaas.conf
+    mode=0644
+  notify:
+    - restart neutron services
```

Comment:
No, it runs as an agent so neutron should already be started for it to be detected.  Removed both restarts.

The lbaas service is restarted when the neutron-linuxbridge-agent restarts.
https://github.com/zrs233/ursula/commit/53d868d61e18b515328c383850c36b25487d85ce#diff-1619285ef9fd3313622cdee1542dee1aR87

This is the same behavior as neutron-dhcp-agent and neutron-l3-agent I believe.
The neutron-linuxbridge-agent gets restarted when "restart neutron services" is called, so that also means the lbaas service gets restarted also doesn't it? 

I see neutron-dhcp-agent and the neutron-l3-agent listed in both "restart neutron services" and they both pass neutron_data_path_agent (neutron-linuxbridge-agent in our case) to upstart_service.  Is this redundancy intentional?  Or do the "start_on" and "stop_on" parameters of upstart_service not work the way I think they do?